src/fadec/README.md

```diff
@@ -0,0 +1,9 @@
+# Engine & FADEC - A32NX/ A380X Build Options
+
+By default, the FADEC code will be built for the A32NX aircraft. To build for the A380X, you will have to edit the build.sh file accordingly:
+
+A380X Option:
+Line 43:   "${DIR}/a380_fadec/src/FadecGauge.cpp"
```

Comment:
Maybe it would be useful to parameterize this in the build script, so they can be both built by CI in the future?

Reply:
Agreed. But for the time being, we have to keep it this way. Once we have an a380 SimObjects source within the same repo, we can build it accordingly.